web/crux/templates/cal-com.json

```diff
@@ -0,0 +1,115 @@
+{
+  "name": "cal.com",
+  "description": "The open-source Calendly alternative. Cal.com is the event-juggling scheduler for everyone. Focus on meeting, not making meetings. Free for individuals. Before deployment make sure to set the 'POSTGRES_PASSWORD' secret and update the 'DATABASE_URL' with the password, as well as set the 'NEXTAUTH_SECRET' and 'CALENDSO_ENCRYPTION_KEY' secrets to a secure random string (16-32 characters should suffice, for example use 'openssl rand -base64 32'). If you'd like to run the app on a port other than 7000, update the exposed port of the 'cal-com' container as well as the 'PORT' and 'NEXT_PUBLIC_WEBAPP_URL' environment variables.",
```

Comment:
Filling ingress name + host and removing external ports are the wae for exposing it to a public domain, that requires traefik. These would be nice to have.

Reply:
Removed ports, added ingress settings. Also modified the description so it shows how to change the domain as well as it tells the user that the template requires Traefik from the agent :)